unity-renderer/Assets/Rendering/Shaders/URP/SceneBoundaries.hlsl

```diff
@@ -0,0 +1,29 @@
+#ifndef SCENE_BOUNDARIES_FILTER
+#define SCENE_BOUNDARIES_FILTER
+    bool SceneBoundaries_IsInParcel(float3 position, half4 parcel)
+    {
+        const half size = parcel.w;
```

Comment:
please always keep this as a parameter! ;)

Also, does it consider global scenes? to not do the check at all

Reply:
It's just a proof of concert, I should have stated that haha. The size is encoded in the parcels array